File: mario-more/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int height;
    do
    {
        height = get_int("Height: ");
    }
    while (height <= 0 || height >= 9);

    for (int i = 1; i <= height; i++)
    {
        for (int d = 1; d <= height - i; d++)
        {
            printf(" ");
        }
        for (int x = 1; x <= i; x++)
        {
            printf("#");
        }
        printf("  ");
        for (int x = 1; x <= i; x++)
        {
            printf("#");
        }
        printf("\n");
    }
}

File: Week06/sentimental-mario-more/mario.py
# TODO
from cs50 import get_int

while True:
    height = get_int("Height: ")
    if height > 8 or height < 1:
        continue
    else:
        break
for i in range(height):
    print((" " * (height - (i + 1))) + ("#" * (i + 1)) + "  " + ("#" * (i + 1)))
